src/Text/Pandoc/Readers/LaTeX.hs

```diff
@@ -691,6 +692,12 @@ inlineCommands = M.fromList $
   , "pagebreak"
   ]
 
+ref :: PandocMonad m => LP m Inlines
+ref = do
+    unlessParseRaw >> do
+        label <- braced
+        return $ spanWith ("",[],[("data-reference", label)]) $ inBrackets $ str label
+
```

Comment:
I added the attribute `data-reference-type` which can contain this information. What do you think about this? 

I will add support for `\pageref`, `\Cref`, etc. later.